apprentice-react-o-shop/src/components/NavBar/dropdownList.js

```diff
@@ -0,0 +1,16 @@
+const dropdownArray = [
```

Comment:
move all constants like this to a constants file (they can all live in one constants file if you want or you can break a folder down into various files your choice)